src/commands/show.ts

```diff
@@ -120,6 +118,12 @@ async function showProblemInternal(node: IProblem): Promise<void> {
         await Promise.all([
             vscode.window.showTextDocument(vscode.Uri.file(filePath), { preview: false, viewColumn: vscode.ViewColumn.One }),
             movePreviewAsideIfNeeded(node),
+            promptHintMessage(
+                "hint.commentDescription",
+                'You can generate problem description as comment in the solution code file by enabling "leetcode.showCommentDescription".',
```

Comment:
You can generate the code file with problem description in the comments by .....